docs/References.md

```diff
@@ -23,6 +23,6 @@ The following artifacts influence and/or describe the Consensus implementation.
 [ouroboros-bft]: https://iohk.io/en/research/library/papers/ouroboros-bfta-simple-byzantine-fault-tolerant-consensus-protocol/
 [ouroboros-praos]: https://iohk.io/en/research/library/papers/ouroboros-praosan-adaptively-securesemi-synchronous-proof-of-stake-protocol/
 [ouroboros-genesis]: https://iohk.io/en/research/library/papers/ouroboros-genesiscomposable-proof-of-stake-blockchains-with-dynamic-availability/
-[consensus-report]: https://hydra.iohk.io/job/Cardano/ouroboros-network/native.consensus-docs.x86_64-linux/latest/download/1
-[network-report]: https://hydra.iohk.io/job/Cardano/ouroboros-network/native.network-docs.x86_64-linux/latest/download/1
+[consensus-report]: https://input-output-hk.github.io/ouroboros-network/pdfs/report/report.pdf
+[network-report]: https://input-output-hk.github.io/ouroboros-network/pdfs/network-design/network-design.pdf
```

Comment:
Consider linking to the pre-redirect links:

> Instead of linking directly to the PDF files, I created proxy HTML files which redirect to them such that we can later change the URL without breaking existing links.

https://github.com/input-output-hk/ouroboros-network/pull/4298#issue-1551239458

```suggestion
[consensus-report]: https://input-output-hk.github.io/ouroboros-network/pdfs/report/report.pdf
[network-report]: https://input-output-hk.github.io/ouroboros-network/pdfs/network-design/network-design.pdf
```

Reply:
Done thx and sorry :smile: